src/controllers/RequestController.js

```diff
@@ -66,6 +66,39 @@ class RequestController {
       return HelperMethods.serverError(res);
     }
   }
+
+  /**
+  * Reject a Request
+  * Route: PATCH: /request
+  * @param {object} req - HTTP Request object
+  * @param {object} res - HTTP Response object
+  * @return {res} res - HTTP Response object
+  * @memberof RequestController
+  */
+  static async rejectRequest(req, res) {
+    try {
+      const { role } = req.decoded;
+
+      if (!role === 'Manager') {
+        return HelperMethods.clientError(res, 'You are not authorized to update this request',
+          401);
+      }
+      const requestExist = await Request.findByPk(req.body.id);
+      if (requestExist.dataValues.status === 'OPEN' && requestExist.dataValues.id) {
+        await requestExist.update({ status: 'REJECTED' });
+        return HelperMethods
+          .requestSuccessful(res, {
+            success: true,
+            message: 'Your request is rejected',
```

Comment:
Good job @ChiamakaObitube. Is the manager rejecting his/her request? or is he/she rejecting someone else's request?. Please make your response message reflect what the request performed successfully.

Reply:
Thank you @Daymorelah for the feedback. The manager is rejecting someone else's request.